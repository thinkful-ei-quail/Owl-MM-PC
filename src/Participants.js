import React from 'react';
import participants from './participantsDummy';
import './Participants.css';

function Participants(props) {
    props.map((person, inSession) => {
        if (person.inSession) {
            return (
                <ul className="participantsList">
                    <li>{person.name}</li>
                    <li>{person.avatar}</li>
                    <li>{person.onStage}</li>
                </ul>
            )
        }
    }) 
};





export default Participants;