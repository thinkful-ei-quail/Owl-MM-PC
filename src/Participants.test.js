import React from 'react';
import ReactDOM from 'react-dom';
import Participants from './Participants';
import participants from './participantsDummy';
import renderer from 'react-test-renderer';


it('should generate participants list', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Participants persons = {[{name:"lip", avatar:"imageFile", onStage:'true', inSession:true}]} />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('snapshot of working UI', () => {
    const tree = renderer
        .create(<Participants persons = {[{name:"lip", avatar:"imageFile", onStage:'true', inSession:true}]} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});