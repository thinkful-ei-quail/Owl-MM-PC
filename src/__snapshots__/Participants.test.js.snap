// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of working UI 1`] = `
<ul
  className="participantsList"
>
  <li>
    lip
  </li>
  <li>
    imageFile
  </li>
  <li>
    true
  </li>
</ul>
`;
